client-reactjs/src/components/application/AssetDetailsDialog.js

```diff
@@ -4,6 +4,8 @@ import useFileDetailsForm from '../../hooks/useFileDetailsForm';
 import { useSelector } from "react-redux";
 import { useParams } from "react-router-dom";
 import { Modal, Button } from 'antd/lib';
+import { color } from 'd3-color';
```

Comment:
Is this package required? Looks like it is not being used

Reply:
Don't remember adding it. It is removed now